README.md

```diff
@@ -1,32 +1,48 @@
+| pylint.showNotification | `off`                                                                                                                                  | Controls when notifications are shown by this extension. Accepted values are `onError`, `onWarning`, `always` and `off`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| pylint.lintOnChange     | `false`                                                                                                                                | Enable linting Python files with Pylint as you type.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| pylint.ignorePatterns   | `[]`                                                                                                                                   |  Configure [glob patterns](https://code.visualstudio.com/docs/editor/glob-patterns) to exclude files or folders from being linted with Pylint.                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+
+The following variables are supported for substitution in the `pylint.args`, `pylint.cwd`, `pylint.path`, `pylint.interpreter` and `pylint.ignorePatterns` settings:
+- `${workspaceFolder}`
+- `${workspaceFolder:FolderName}`
+- `${userHome}`
+- `${env:EnvVarName}`
+
+The `pylint.path` setting also supports the `${interpreter}` variable, which is subtituted based on the value of the `pylint.interpreter` setting.
```

Comment:
This also substitutes any python selected using Python extension.

Reply:
only if `python.interpreter` has been updated right?

Reply:
If `pylint.interpreter` then `${intepreter}` is same as that value. If `pylint.interpreter` is not set, then we ask python extension to get the interpreter for that workspace, and `${interpreter`} is same as the value returned by python extension.

`${interpreter` can **not** be mixed with other thing like "${interpreter}/something" is not allowed. but you can do `["${interpreter}", "./something"]`

Reply:
I added a clarification that it accepts as one of the entries in the array and added another example using `["${interpreter}", "-m", "pylint"]`